README.md

```diff
@@ -1,17 +1,44 @@
-# Salable Web Components Svelte Demo
+# Salable Web Components : Svelte Demo
 
-This repository demonstrates how to set up Salable Web Components with Svelte.
-g
-## Getting started
+[Salable](https://www.salable.app) is a subscription licensing platform that allows users to have more control over their customers and how they interact with and use your applications/products.
 
-Clone the repo install the dependencies and run the project
+This demo repository shows how you can implement Salable Web Components into a Svelte project to do things such as:
+
+- Show pricing tables to allow users to subscribe to plans and create licenses
+- Show/Download all of the invoices associated with a subscription
+- Allow users to subscribe to a plan via an in-line checkout
+
+## Running the demo
+
+To run this demo locally with your own Salable account, follow the steps below.
+
+1. Clone this repository (`git clone https://github.com/Salable/Salable-Web-Components-Svelte-Demo.git`)
+2. Install the demo's dependencies (`npm install`)
+3. [Create a Salable account](https://salable.app/signup) (if you don't already have one)
+4. Create the required data for your chosen web component ([See requirements for each component here](https://docs.salable.app/web-components/web-components-latest/))
+5. Create a new session token for your chosen component using the Salable API
```

Comment:
The session token bit needs more information and/or a link to the api docs.

Reply:
At least link to the API docs and make a ticket to improve this for all readmes later.